### PR TITLE
netdata/packaging: Split CUPS plugin to separate package (RPM)

### LINKED ISSUE #6662: ubuntu: make cups package optional
At the moment the Ubuntu netdata package has cups as dependency. Normally cups is not required on server systems. Please mark cups as suggested package.

### PR DESCRIPTION
**Pull request by paulkatsoulakis**
##### Summary
As per the #6662 report, the time has come to consider separating all plugins that are easy to split on separate packages. 

This is the first plugin from a series of plugins that can be separated.
Once we fine grain the results of this PR we will mass-change the rest of the plugins following the same templated way given here.

Note: won't be touching .DEB sub-packaging yet as i need more context to get through

##### Component Name
netdata/packaging

##### Additional Information
Fixes #6662

### CODE REVIEW

**Comment by Ferroin** [comment]
If we're planning on keeping the plugins in the main repo, it would be helpful to be able to manually build each plugin individually, independent of whether or not the core is being built (and it would be similarly helpful to be able to build the core by itself).  That would greatly simplify having split packages for source-based distributions, as well as making life easier for anybody who wants to build packages locally.

**Comment by paulkatsoulakis** [comment]
> packaging/version should not be updated in PRs

A `git rebase master` aligned it. It was simply because of the branch be left behind

File: .travis.yml [hunk 1]
```diff
@@ -27,9 +27,10 @@ install:
   - export LATEST_RELEASE_DATE="$(git log -1 --format=%aD "${LATEST_RELEASE_VERSION}" | cat)"
   - if [[ "${TRAVIS_COMMIT_MESSAGE}" = *"[Build latest]"* ]]; then export BUILD_VERSION="$(cat packaging/version | cut -d'-' -f1,2 | sed -e 's/-/./g').latest"; fi;
   - export DEPLOY_REPO="netdata"  # Default production packaging repository
+  - export PACKAGING_USER="netdata" # Standard package cloud account
   - if [[ "${TRAVIS_COMMIT_MESSAGE}" = *"[Build latest]"* ]]; then export DEPLOY_REPO="netdata-edge"; fi;
-  - export PACKAGING_USER="$(echo ${TRAVIS_REPO_SLUG} | cut -d'/' -f1)"
   - export PACKAGE_CLOUD_RETENTION_DAYS=30
+  - if [ "${TRAVIS_REPO_SLUG}" = "paulkatsoulakis/netdata" ]; then export DEPLOY_REPO="netdata-devel";  fi;
```
Comment (cakrit) [inline .travis.yml]: No, never.

**Comment by thiagoftsm** [comment]
Hi @paulkatsoulakis ,

Is the PR ready to be tested again?

**Pull request merged**